packages/connect-common/files/coins.json

```diff
@@ -2228,6 +2228,26 @@
     "erc20": [],
     "eth": [],
     "misc": [
+        {
+            "blockchain_link": {
+                "type": "solana",
+                "url": [
+                    "wiser-green-owl.solana-mainnet.discover.quiknode.pro/c218d0c9e451f6c5c0bf8b7c7d8e5a72384c9570/"
```

Comment:
FYI: This a temporary url, just my personal quicknode account so we have something to work with.